docs/content/src/develop/setup-dev.md

```diff
@@ -1,20 +1,44 @@
 
-## Setting up IDEs
+![alt text](../assets/img/vs-code-plugin.png)
 
-You can use your [favorite editor to work on Ballerina code](https://ballerina.io/learn/tools-ides/), however, we recommend you use VS Code as we have made some improvements to it for use in integration scenarios.
+Once installed, the following `Ballerina Integrator Dashboard` shown below could be used for creating new Ballerina
+projects, creating modules using available module templates and browsing the available module templates. The sequence
+diagram of Ballerina code could be viewed using the `Ballerina: Show File Overview` command. Refer the Ballerina
+Integrator VS Code extension's [home page](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina-integrator) for
+more instructions on using the extension.
 
-Download and install [VS Code from the Visual Studio website](https://code.visualstudio.com/Download) if you do not have it already.
+![alt text](../assets/img/vs-code-landing.png)
 
-Find the extension for Ballerina in the VS Code marketplace. This extension is called `Ballerina`. For instructions on installing and using it, see [The Visual Studio Code Extension](https://ballerina.io/learn/tools-ides/vscode-plugin/).
+## Installing Ballerina Integrator via Installers
 
-![alt text](../../assets/img/vs-code-ballerina-plugin.png)
+### Installing on OS X
 
-A Quick Start Guide is available along with the plugin that helps with installing it. Once installed, the following page appears.
+Download the package file and double-click on it to launch the installer. The installer guides you through the installation process and installs the Ballerina distribution to `/Library/Ballerina`.
 
-![alt text](../../assets/img/vs-code-landing.png)
+The package automatically sets your PATH environment variable for you. You may need to restart any open Terminal sessions for the change to take effect.
 
+### Installing on Windows
+
+Download the MSI file and double-click on it to launch the installer. The installer guides you through the installation process and installs the Ballerina distribution to `C:\Program Files\Ballerina\`.
+
+The installer should put the `C:\Program Files\Ballerina\<ballerina-directory>\bin` directory in your PATH environment variable. You may have to restart any open command prompts for the change to take effect.
+
+### Installing on Linux
+
+Download the DEB file, use the following command to install the Ballerina Integrator. Replace `<ballerina-home>/<ballerina-binary>.deb` with the actual file path. This installs the Ballerina Integrator in the `/usr/lib/ballerina` directory.
+```
+dpkg -i <ballerina-binary>.deb
+```
+
+### What's Next
+Once you have successfully installed Ballerina, try out the [Quick Start Guide](../get-started/quick-start-guide.md) and take Ballerina Integrator for its first spin.
```

Comment:
Did you test this link using mkdocs serve?

Reply:
Yes, it works.